c/Makefile

```diff
@@ -24,7 +20,5 @@ distclean: clean
 clang.db: /usr/lib/stdlib.repo
 	cp $< $@
 
-CRTI = $(MUSL)/lib/crt1.t $(MUSL)/lib/crt1_asm.t
-
 a.out: main.o
-	rld -o $@ $(CRTI) main.o $(MUSL)/lib/libc_repo.a
+	$(CC) -target x86_64-pc-linux-musl-repo main.o -o $@
```

Comment:
I'm being picky, but, to my eyes, moving the `-o $@` to the end of the command line (after the positional arguments) looks weird and is different to the other makefiles. Could we restore the order?

Reply:
Thanks, fixed in the commit [0eadd6e8 ](https://github.com/SNSystems/hello-rld/pull/3/commits/0eadd6e800e4e8589ae26c2296b1ff5a20dc44d0).